src/renderer/src/components/exchange/Exchange.tsx

```
@@ -1,6 +1,8 @@
 import { ReactElement, useEffect } from 'react';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
 import { useDispatch, useSelector } from 'react-redux';
-import { Link as RouterLink, useParams } from 'react-router-dom';
+import { Link as RouterLink, useNavigate, useParams } from 'react-router-dom';
```

ESLint

[error] 'useNavigate' is defined but never used.

(@typescript-eslint/no-unused-vars)